src/main/java/com/sss/garage/data/driver/DriverData.java

```diff
@@ -1,8 +1,15 @@
 package com.sss.garage.data.driver;
 
+import com.sss.garage.model.elo.Elo;
+
+import java.util.Set;
+
 public class DriverData {
     private Long id;
     private String nickname;
+    private Set<Elo> elos;
```

Comment:
W Data object albo primitive type albo inne data object, na pewno nie encje. Gdybys mial jakies elo na driverze i puscil requesta po detailed driver to by sie zrobila nieskonczona petla. Czyli trzeba zrobić EloData